src/libsyntax_pos/hygiene.rs

```diff
@@ -575,9 +550,15 @@ impl Span {
     /// The returned span belongs to the created expansion and has the new properties,
     /// but its location is inherited from the current span.
     pub fn fresh_expansion(self, expn_data: ExpnData) -> Span {
+        self.fresh_expansion_with_transparency(expn_data, Transparency::SemiTransparent)
```

Comment:
Does `Transparency::Transparent` makes more sense here? This expansion isn't for a macro so it can't be used for hygienic resolution anyway.

Reply:
Some of the things using `fresh_expansion` currently use `gensym`s too, so it may be better to organize opaque expansions for them.
`Transparency::SemiTransparent` is used here just to keep the pre-existing behavior for now.